### ISSUE TITLE
not working anymore since new updates

### ISSUE THREAD

**Issue by bwired-nl** [Fleeceable/Nebra-Helium-Watchdog#8]
HI
Its not working anymore since the helium outages and updates

**Comment by bwired-nl** [comment]
update: its working but not continious, looks like its an api problem, not in your code

**Comment by Fleeceable** [comment]
Yeah. Nebra API should be up on monday. I'll probably release another version where Heltec miners are also supported when API is reachable...
